rootfs/api/tests/test_maintenance.py

```diff
@@ -0,0 +1,66 @@
+import requests_mock
```

Comment:
can you rename this file to `test_app_settings.py` since that is the object we're testing here?